quinn-proto/src/connection/stats.rs

```diff
@@ -17,6 +17,25 @@ impl std::fmt::Debug for PacketTransmissionStats {
     }
 }
 
+/// Statistics about packets received on a connection
+#[derive(Default)]
+#[non_exhaustive]
+pub struct PacketReceptionStats {
+    /// The amount of packets transmitted on a connection
+    pub packets: u64,
+    /// The total amount of bytes transmitted on a connection
+    pub bytes: u64,
+}
```

Comment:
Should this and `PacketTransmissionStats` be the same struct, similar to the intended use of `FrameStats`?

Reply:
It could be. I guess a question is how one would deal with stats that might later be different between rx and tx. E.g. congestion/pacing/etc would only apply to tx. They could however be in a different struct.

If they are unified, I guess it would be `PacketStats` or `UdpDatagramStats`?

Reply:
> E.g. congestion/pacing/etc would only apply to tx. They could however be in a different struct.

That sounds good to me.

> If they are unified, I guess it would be PacketStats or UdpDatagramStats?

How about `UdpStats`?